src/database/DataTypes/Item.js

```diff
@@ -299,6 +299,10 @@ export class Item extends Realm.Object {
       minTemperature: sensorLogs.min('temperature'),
     };
   }
+
+  getAllSensorLogs(database) {
+    return database.objects('SensorLog').filtered('itemBatches.item.id = $0', this.id);
+  }
```

Comment:
Do we need to check for itemBatches.numberOfPacks not zero here?

Reply:
fabcdb468a1da7cae9d7fe9fd6501e19f0eee28f